reference/forms/types/collection.rst

```diff
@@ -16,10 +16,10 @@ forms, which is useful when creating forms that expose one-to-many relationships
 | Options     | - `allow_add`_                                                              |
 |             | - `allow_delete`_                                                           |
 |             | - `delete_empty`_                                                           |
-|             | - `options`_                                                                |
+|             | - `entry_options`_                                                                |
```

Comment:
Shouldn't we keep the old options marking them as deprecated? Removing them entirely can be confusing for people coming from older versions.